applications/ChimeraApplication/CMakeLists.txt

```diff
@@ -0,0 +1,86 @@
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
+
+message("**** configuring KratosChimeraApplication****")
+
+
+## Check if fluid dynamics app is being compile dor not
+if(${FLUID_DYNAMICS_APPLICATION} MATCHES OFF )
```

Comment:
this does no longer work, I recommend checking the RANS-App to see how it works now